source/MaterialXTest/MaterialXRenderOsl/GenReference.cpp

```diff
@@ -63,7 +63,7 @@ TEST_CASE("GenReference: OSL Reference", "[genreference]")
     // Ignore the following nodes:
     const mx::StringSet ignoreNodeList = { "surfacematerial", "volumematerial",
                                            "constant_filename", "arrayappend",
-                                           "dot_filename"};
+                                           "dot_filename", "convert_BSDF_shader", "convert_EDF_shader", };
```

Comment:
Was this left here by mistake? As far as I can see there are no convert nodes for BSDF or EDF yet.

Reply:
I'll remove this. I was trying to get BSDF and EDF convert to work but there are issues and was removed.